src/kaltura-player.js

```diff
@@ -22,6 +23,7 @@ export default class KalturaPlayer {
   constructor(options: KalturaPlayerOptionsObject) {
     this._player = loadPlayer(options.player);
     this._logger = getLogger('KalturaPlayer' + Utils.Generator.uniqueId(5));
+    configureJsonp(this._player._config);
```

Comment:
1. You are accessing a private member - you can't do it.
2. If you want to manipulate the configuration of the player you need to move the `configureJsonp` to the setup workflow, before the kaltura player has been created.